builds/checkin/libiothsm.yaml

```diff
@@ -4,16 +4,30 @@ pr:
     include:
       - master
       - release/*
-  paths:
-    include:
-      - builds/*
-      - edgelet/*
 jobs:
 
+################################################################################
+  - job: check_run_pipeline
```

Comment:
It would be really great to make this a template so we don't have three copies

Reply:
I suggest we test this setup as is and when it is proven to work we make improvements. How that sounds to you guys?

Reply:
In addition, this step is not fully identical for all pipelines because of how egrep command search for regex pattern, but hopefully can be done.

Reply:
Yes the egrep pattern will need to be an input variable. I'm fine with doing the refactoring later.